lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -96,6 +96,20 @@ if (UFE_LIGHTS_SUPPORT)
     )
 endif()
 
+message(STATUS "UFE_MATERIALS_SUPPORT is ${UFE_MATERIALS_SUPPORT}")
```

Comment:
This printf might not be necessary anymore.

Reply:
Correct, the message is output by FindUFE.cmake

Reply:
Might be a good idea if you have the time.